src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -414,12 +431,15 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
   // Global actions
   @Action setOfficeAddresses!: ActionBindingIF
   @Action setEditingOfficeAddresses!: ActionBindingIF
+  @Action setValidComponent!: ActionBindingIF
 
   // Declarations for template
   readonly isEmpty = isEmpty
   readonly AddressTypes = AddressTypes
   readonly CorpTypeCd = CorpTypeCd
   readonly addressSchema = OfficeAddressSchema
+  readonly RegistrationMailingAddressSchema = PersonAddressSchema
+  readonly RegistrationDeliveryAddressSchema = OfficeAddressSchema
```

Comment:
I like this aliasing here.

Now I see that _some time_ (future). we should rename PersonAddressSchema and OfficeAddressSchema -- for example, the person one isn't being use for a person so...

Future.

Reply:
Agreed!

Reply:
Haha, I think I made that comment once already.

It's a double-edged sword. It can be used to self-document what you're doing (like here). So I like this. But it seems weird to use a "person" schema" for a "registration mailing address". Ideally a reviewer (or future developer, or me after I forget) would be able to guess what the schema contains based on its name.

Still future 😁